src/prommis/nanofiltration/tests/test_nf_brine.py

```diff
@@ -0,0 +1,48 @@
+#################################################################################
+# adapted from test_nf.py
+#################################################################################
+
+"""
+Tests the nanofiltration flowsheet
+"""
+
+from pyomo.environ import value
+
+import pytest
+
+pytest.importorskip("watertap", reason="WaterTAP dependency not available")
+# pylint: disable-next=wrong-import-position
+from prommis.nanofiltration.nf_brine import main
+
+
+@pytest.mark.requires_idaes_solver
```

Comment:
Just to check - does this mark exist? I have not seen it before, but it might be a PrOMMiS specific mark (although I do not see a pytest.ini file to declare them).

Reply:
Good point, this is a marker from WaterTAP where we run pytest with non-IDAES Ipopt as part of the CI checks. Since that's not the case here, I suggest to remove it.

```suggestion
```